### ISSUE TITLE
[Issue]: NVM for Windows is INCORRECTLY positioned in the PATH

### ISSUE THREAD

**Issue by metal450** [coreybutler/nvm-windows#1056]
### What happened?

* Uninstalled previous installation of Node
* Developer Mode enabled
* Installed Nvm 
* In an administrator prompt, `nvm install latest`
* `nvm use latest` -> shows `Now using node v21.1.0 (64-bit)`
* However, `node` doesn't work (not a recognized command).
* I tried the "pathfinder" utility, which reports:

```
PATH directories containing node.exe:
NVM for Windows is INCORRECTLY positioned in the PATH (must be first)
A prior/alternative installation of Node.js may be preventing NVM for Windows from functioning.
```

After getting this error the first time, I tried rebooting & reinstalling nvm again, but the error was the same. The previous version of node has definitely been unistalled.

### What did you expect to happen?

`node` to work, or `nvm` to show some explanatory error about the issue

### Version

1.1.11 or newer (Default)

### Which version of Windows?

Windows 10 (Default)

### Which locale?

English (Default)

### Which shell are you running NVM4W in?

Command Prompt

### User Permissions?

Administrative Privileges, Elevated

### Is Developer Mode enabled?

Yes

### Debug Output

```Shell
Running NVM for Windows with administrator privileges.

Administrator: C:\Windows\System32\cmd.exe - nvm  debug
NVM4W Version:      1.1.11
NVM4W Path:         C:\Users\Metal450\AppData\Roaming\nvm\nvm.exe
NVM4W Settings:     C:\Users\Metal450\AppData\Roaming\nvm\settings.txt
NVM_HOME:           C:\Users\Metal450\AppData\Roaming\nvm
NVM_SYMLINK:        C:\Program Files\nodejs
Node Installations: C:\Users\Metal450\AppData\Roaming\nvm

Active Node.js Version: none
(run "nvm use <version>" to activate a version)

PROBLEMS DETECTED
-----------------
"Administrator: C:\Windows\System32\cmd.exe - nvm  debug" is not an officially supported shell. Some features may not work as expected.



Find help at https://github.com/coreybutler/nvm-windows/wiki/Common-Issues
```

**Comment by metal450** [comment]
...And right after posting, I found the issue.

Although the previous version of node *had been* uninstalled, there was still a leftover C:\Program Files\nodejs folder (containing just node_modules/npm/node_modules/node-gyp/...). After removing it manually, it worked.

So the solution is: nvm needs to report an error if there's an existing nodejs folder there, which isn't a symlink. Currently it just purports to have successfully set the current version, which it has not.

**Comment by coreybutler** [comment]
`nvm debug` already reports this to users.

**Comment by metal450** [comment]
Huh? Above is the output of `nvm debug`, it doesn't say anything about this (the only thing under PROBLEMS DETECTED is `"Administrator: C:\Windows\System32\cmd.exe - nvm  debug" is not an officially supported shell. Some features may not work as expected.`, which I was unclear about what it meant)

**Comment by coreybutler** [comment]
I'm sorry, I wasn't clear at all! I've already added support for this in `nvm debug` in the main branch. It hasn't been released yet, but it should be available in 1.1.12 when it's released. 

Hopefully that false positive problem you're seeing will go away too. Every version of Windows seems to have a different title on the command prompts, which makes it hard to accurately determine which program is actually running. I've added some checks for things like "cmd.exe", which should resolve that issue.
